my_enumerable_module.rb

```diff
@@ -0,0 +1,72 @@
+module Enumerable
```

Comment:
I see this was part of the project, but please note, that opening an existing module or class this way and adding things while doable and sometimes a good idea, can often lead to a very bad place lol.

Reply:
lol good to know! do you have suggestions for what i could change it to? maybe i can do something like 

'''
class MyEnumerable
  include Enumerable 
'''
but that would require me to also change all my method names right?

Reply:
Oh ... this is fine for the project. In real life though, you'd want to think long and hard about opening up something like this and monkey patching (term for changing an existing module class). I will note, that Rails, does this quite a bit, but DHH is probably a bit smarter than us lol!

Reply:
Ah, gotcha. Haha he definitely knows way more than me that's for sure 😂